app/api/v2/handlers/agent_api.py

```diff
@@ -59,26 +59,32 @@ async def update_agent(self, request: web.Request):
         agent = await self.update_object(request)
         return web.json_response(agent.display)
 
-    @aiohttp_apispec.docs(tags=['agents'])
+    @aiohttp_apispec.docs(tags=['agents'], summary="Update an agent, or if a existing agent match cannot be found, "
```

Comment:
The `summary` key is essentially a title for these endpoints. For each endpoint, can you add a `summary` and `description` for the `@aiohttp_apispec.docs`?

I think for these endpoints, what is currently the `summary` is a bit more like the `description` (i.e. a more detailed writeup about the endpoint), and the `summary` can just be something like "Replace an Agent."

Reply:
Got it - didn't know description was a field.